app/static/src/app/components/options/TagInput.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <vue-tags-input style="border-color: #d7dce1;"
+                    :tags="computedTags"
+                    :placeholder="'...'"
+                    :validate="validate"
+                    @on-tags-changed="handleTagsChanged"/>
+</template>
+<script lang="ts">
+import {
+    PropType, computed, defineComponent
+} from "vue";
+import VueTagsInput from "vue3-tags-input";
+import { useStore } from "vuex";
+import { Tag } from "../../store/run/state";
+
+export default defineComponent({
+    components: {
+        VueTagsInput
+    },
+    props: {
+        tags: Array as PropType<Tag[] | null>
+    },
+    setup(props, { emit }) {
+        const store = useStore();
+
+        const paramValues = computed(() => store.state.run.parameterValues);
+
+        const computedTags = computed(() => {
+            if (props.tags?.length === 0 || !props.tags) {
+                return [];
+            }
+            const parsedTags = props.tags.map((tag) => {
+                if (typeof tag === "number") {
+                    return `${tag}`;
+                }
+                const tagValue = paramValues.value[tag];
+                if (tagValue !== undefined) {
+                    return `${tag}: ${tagValue}`;
+                }
+                return undefined;
+            });
+            return parsedTags.filter((x) => x !== undefined);
+        });
+
+        const isNumeric = (num: any) => {
```

Comment:
Is this not always a string?

```suggestion
        const isNumeric = (value: string) => {
```

Reply:
it is always string! i have changed it, i think the reason i kept it as any is because it true does check any, like from a completely local point of view haha, but yh changed it!